setup.sh

```diff
@@ -1,19 +1,22 @@
 #! /bin/sh
 
+# make script fail on errors
+# TODO add exit trap to report the failing line? at least say there was an error
+set -eu
+IFS=$'\n\t'
 ##
 # Wrapper for various setup scripts included in the docker-mailserver
 #
 
-INFO=$(docker ps \
-  --no-trunc \
-  --format="{{.Image}}\t{{.Names}}\t{{.Command}}" | \
-  grep "supervisord -c /etc/supervisor/supervisord.conf")
-
-IMAGE_NAME=$(echo $INFO | awk '{print $1}')
-CONTAINER_NAME=$(echo $INFO | awk '{print $2}')
+IMAGE_NAME=tvial/docker-mailserver:latest
```

Comment:
Hard-coding the image may not be a good idea as there can be changes between latest, stable and versioned releases. We should ideally use the same version that the user intends to run (or is running). As a default image name this works fine.

Reply:
Yes, I totally agree with that. In the end the script should check if there is any running container that is based on tvial/docker-mailserver.

Additionally the script has (at the moment) the ability to specify the name of the running container.

Reply:
That feature (specifying container name) is used extensively from the tests and is important. It is probably less important in real life, in theory someone could run multiple docker images on the same server but that is bound to be exceedingly rare.